### PR TITLE
fix css img

### LINKED ISSUE #38: fix css

### PR DESCRIPTION
**Pull request by axelcasa**

### CODE REVIEW

File: src/pages/Partners/index.tsx [hunk 1]
```diff
@@ -173,44 +174,45 @@ const Partners: React.FC = () => {
...
+                        <Card sx={{border: '1px solid lightgrey'}} elevation={0}>
+                            <CardActionArea onClick={
+                                () => window.open("https://icosys.ch/", "_blank")
+                            }>
+                                <CardMedia
+                                    sx={{p: 2}}
+                                    component="img"
+                                    height="200px"
+                                    image="/icosys-logo.png"
+                                    alt="ALP ICT Logo"
```
Comment (andrptrc) [inline src/pages/Partners/index.tsx]: Wrong alt

File: src/pages/Partners/index.tsx [hunk 2]
```diff
@@ -288,42 +290,44 @@ const Partners: React.FC = () => {
...
-                    </Card>
-                </Box>
+                        <Card elevation={0} sx={{border: '1px solid lightgrey'}}>
+                            <CardActionArea onClick={
+                                () => window.open("https://www.arcinfo.ch/", "_blank")
+                            }>
+                                <CardMedia
+                                    sx={{p: 1}}
+                                    component="img"
+                                    height="100px"
```
Comment (andrptrc) [inline src/pages/Partners/index.tsx:323]: 200px to be consistent every where ?

**Review by andrptrc — CHANGES_REQUESTED** [review]

File: src/pages/Partners/index.tsx (hunk 2 above)
Comment (axelcasa) [reply src/pages/Partners/index.tsx:323]: no, the logo is too big

Comment (andrptrc) [reply src/pages/Partners/index.tsx:323]: ok

**Review by andrptrc — APPROVED** [review]

**Pull request merged**